Projects/Team3/Andrii Nazaruk/HomeWork2.2/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace HomeWork2._2
+{
+    class Program
+    {
+        //2. Дано натуральне число а(a<100). Напишіть програму, що виводить на екран кількість цифр в цьому числі і суму цих цифр
+        static void Main(string[] args)
+        {
+            Random r = new Random();
+            int Digit = r.Next(0, 99);
+            Console.WriteLine("a = "+Digit);
+            int firstDigit = Digit/10;
+            int secondDigit = Digit % 10;
+            int count;
+            int summ;
+            
+            if(Digit>10 && Digit<100)
+            {
+                count = 2;
+                summ = firstDigit + secondDigit;
+                Console.WriteLine("FirstDigit = " + firstDigit);
+                Console.WriteLine("FirstDigit = " + secondDigit);
+                Console.WriteLine("Count : " +count);
+                Console.WriteLine("Summ = " +summ);
+            }
+            else
+            {
+                Console.WriteLine("Digit <100");
+            }
```

Comment:
will your code handle a case when 0< digit <10 ?

Reply:
All right, I will do it again.